--- a/src/qemu/qemu_capabilities.c.orig	2015-04-28 02:15:40.000000000 +0100
+++ a/src/qemu/qemu_capabilities.c	2015-05-15 09:58:45.948885821 +0100
@@ -3331,6 +3331,7 @@
      * daemonize guarantees control won't return to libvirt
      * until the socket is present.
      */
+    /* Remove daemonize since this hangs qemu 1.6 */
     cmd = virCommandNewArgList(qemuCaps->binary,
                                "-S",
                                "-no-user-config",
@@ -3339,12 +3340,13 @@
                                "-M", "none",
                                "-qmp", monarg,
                                "-pidfile", pidfile,
-                               "-daemonize",
                                NULL);
     virCommandAddEnvPassCommon(cmd);
     virCommandClearCaps(cmd);
     virCommandSetGID(cmd, runGid);
     virCommandSetUID(cmd, runUid);
+    virCommandSetPidFile(cmd, pidfile);
+    virCommandDaemonize(cmd);
 
     virCommandSetErrorBuffer(cmd, qmperr);
 
