diff -Nurp libvirt-1.0.1/configure.ac libvirt-1.0.1.fix/configure.ac
--- libvirt-1.0.1/configure.ac	2012-12-17 05:14:50.000000000 +0200
+++ libvirt-1.0.1.fix/configure.ac	2013-01-14 20:55:43.746989140 +0200
@@ -959,7 +959,9 @@ dnl
 if test "$with_linux" = "yes"; then
   if test "$with_qemu" = "yes" || test "$with_lxc" = "yes" ; then
     AC_CHECK_HEADERS([linux/param.h linux/sockios.h linux/if_bridge.h linux/if_tun.h],,
-                     AC_MSG_ERROR([You must install kernel-headers in order to compile libvirt with QEMU or LXC support]))
+	[AC_MSG_ERROR([You must install kernel-headers in order to compile libvirt with QEMU or LXC support])],
+	[[#include <netinet/in.h>
+	]])
   fi
 fi
 
diff -Nurp libvirt-1.0.1/src/util/virnetdevbridge.c libvirt-1.0.1.fix/src/util/virnetdevbridge.c
--- libvirt-1.0.1/src/util/virnetdevbridge.c	2012-09-21 05:02:13.000000000 +0300
+++ libvirt-1.0.1.fix/src/util/virnetdevbridge.c	2013-01-14 21:25:57.954062079 +0200
@@ -32,6 +32,7 @@
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 #include <net/if.h>
+#include <netinet/in.h>
 
 #ifdef __linux__
 # include <linux/sockios.h>
